DQMOffline/Trigger/python/HigPhotonJetHLTOfflineSource_cfi.py

```diff
@@ -28,7 +28,7 @@
     dirname = cms.untracked.string("HLT/Higgs/PhotonJet"), 
     verbose = cms.untracked.bool(False), # default: False
     triggerAccept = cms.untracked.bool(True), # default: True 
-    triggerResultsToken = cms.InputTag("TriggerResults"),
+    triggerResultsToken = cms.InputTag("TriggerResults","","HLT"),
```

Comment:
As an aside, the code I'm now debugging will run the above example correctly. It will identify that `process.analyzer` needs `process.producer` and will have `process.path1` wait until `process.path2` has run (or failed to run) `process.producer` before executing `process.analyzer`.
Such intra-path dependencies are needed since we build `DIGI` and `RECO` as seperate paths instead of as unified paths and therefore the framework has to schedule them correctly.